source/index.css

```diff
@@ -20,4 +20,40 @@ canvas {
     @include aspect-ratio(160, 90);
 }
 
+.player-score {
+    position: relative;
+    text-align: center;
+    text-decoration: underline;
+    font-weight: bold;
+}
+
+.player-score .foreground-score {
+    position: relative;
+    z-index: 1;
+}
+
+.player-score .shadow-score {
```

Comment:
I forgot CSS3 supports this and for once in my life I'm not supporting legacy browsers, I'll change this in another commit.